app/inputs/select_with_modal_help_input.rb

```diff
@@ -31,6 +31,13 @@ def build_field(value, _index)
       @rendered_first_element = true
 
       html_options.merge!(options.slice(:include_blank))
-      template.select_tag(attribute_name, template.options_for_select(select_options, value), html_options)
+      render_options = select_options
+      if value.present? && render_options.find { |opt| opt[1] == value }.nil?
+        force_option = RightsService.select_inactive_options.find { |opt| opt[1] == value } || [value, value]
```

Comment:
I'm wondering about factoring lines 35 and 36 into a service method? Two sets of find in view logic is a bit of a warning flag for me. There is also an assumption about the found results (i.e. `opt[1]` implies we get back an array).

One option I had thought of was:

```ruby
RightsService.with_forced_label_and_value_for(render_options, value) do | label, value |
  html_options[:class] << "force-select"
  html_options[:'data-force-label'] = label
  html_options[:'data-force-value'] = value
end
```

I don't know if that is more clarifying or less from a view legibility stand point?